reference/artifacts-with-artifactsrewrite/types/docker-image/index.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: single
+title:  "Docker Image"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+A Docker image is a snapshot of a container, to be run locally or in the
+cloud. Docker image artifacts are used as references to images in
+[registries](https://docs.docker.com/registry/), such as
+[Google Cloud Container Registry](https://cloud.google.com/container-registry/)
```

Comment:
I think the preferred name is "Google Container Registry" (without the "Cloud") but @dorbin may know better.